dev-python/bitarray/Manifest

```diff
@@ -1,5 +1,2 @@
 DIST bitarray-1.7.1.tar.gz 58441 BLAKE2B f09fa3ffe2cc6534984e9da05ddff767dcca59075f5f08977237eab592a8e3e28ebec2d2bde89a21bba14f4c59a5e0411be50abfe562ebfbaaeca01339786dec SHA512 f45461699f5e48b97001c64c514cdafe208fe31f7cf6c211aa0fb2f32d46028717fcbeb38b812bc8f349226537a3ec6dfec97bc7ecf7253d484332307504abd0
```

Comment:
Do not remove potential stable candidates.

Reply:
Ah, sorry, missed it in the process